src/main/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseGroupEndpoint.java

```diff
@@ -25,33 +38,116 @@
 @Slf4j
 public final class CaseGroupEndpoint implements CTPEndpoint {
 
-  public static final String ERRORMSG_CASEGROUPNOTFOUND = "CaseGroup not found for";
-
-  @Autowired
-  private CaseGroupService caseGroupService;
-
-  @Qualifier("caseSvcBeanMapper")
-  @Autowired
-  private MapperFacade mapperFacade;
-
- /**
-   * the GET endpoint to find CaseGroups by caseGroupId
-   *
-   * @param caseGroupId UUID to find by
-   * @return the casegroups found
-   * @throws CTPException something went wrong
-   */
-  @RequestMapping(value = "/{caseGroupId}", method = RequestMethod.GET)
-  public CaseGroupDTO findCaseGroupById(@PathVariable("caseGroupId") final UUID caseGroupId)  throws CTPException {
-    log.info("Entering findCaseGroupById with {}", caseGroupId);
-
-    CaseGroup caseGroupObj = caseGroupService.findCaseGroupById(caseGroupId);
-    if (caseGroupObj == null) {
-        throw new CTPException(CTPException.Fault.RESOURCE_NOT_FOUND, String.format("%s casegroup id %s",
-                ERRORMSG_CASEGROUPNOTFOUND, caseGroupId));
+    @Autowired
```

Comment:
Reformatted the file. Only added a couple of services and the methods findTransitionsByCollectionExerciseIdAndRuRef and changeCaseGroupStatus